index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id="title"> survey form for microverse </h1>
+<P id="description">This is a test for the trials</P> 
+
+<div>
+<form id="survey-form">
+
+  <label id="name-label">Name:</label><br>
+
+  <input type="text" id="name" required placeholder="Write your name"><br>
+
+  <label id="email-label">Email</label><br>
+
+  <input type="email" id="email" required placeholder="Write your email"><br><br>
+
+  <label id="number-label" >Age</label><br>
+
+  <input type="number" id="number" min="18" max="99" placeholder="write your age"><br><br>

```

Comment:
♻️ Good thing you used the correct HTML forms when creating your form! 🔥

Remember, though, that it's better to style elements with CSS rather than using multiple <br> tags. Consider using CSS for this purpose.

You can use ::after for that.